### DISCUSSION TITLE
How can I prevent the API path and Bearer Token from being seen on the Network?

### DISCUSSION THREAD

**Discussion by juanfh** [vercel/next.js#35509] [Help]
UPDATED

**Comment by GabenGar** [comment]
Where do `APIROUTE` and `APITOKEN` values come from?

**Reply by GabenGar** [reply]
They should be `undefined` on the client then. Unless you prefixed them with `NEXT_PUBLIC_`.

**Reply by GabenGar** [reply]
Well yeah you can't make "hidden" ajax calls from the client script. In fact nothing is hidden client-side so you shouldn't have any persistent private keys in the interactions between server and client.
